util/changelog.rb

```diff
@@ -56,13 +56,7 @@ def initialize(file, version)
     @version = Gem::Version.new(version)
     @file = File.expand_path(file)
     @config = YAML.load_file("#{File.dirname(file)}/.changelog.yml")
-    @level = if @version.segments[1..2] == [0, 0]
-               :major
-             elsif @version.segments[2] == 0
-               :minor
-             else
-               :patch
-             end
+    @level = @version.segments[2] != 0 ? :patch : :minor
```

Comment:
Does this actually support majors, like the PR title says? (It's fine if not, just wanted to clarify for myself so I would know)

Reply:
Basically, as this is coded, there's no difference between major and minor, I should've named the level `:minor_or_major` to make this clear. We allow any types of changes in majors and minors, the only difference is subjective, major should include major breaking changes, but minors are allowed to ship small breaking changes.

I did this because we have a few changes in rubygems that can be considered breaking, but they affect small unimportant cases in my opinion. Check the changelog, if we don't agree to ship them now, I think the easiest would be to revert them from the main branch.

Reply:
Works for me, thanks for explaining! 👍🏻

Reply:
Great, I'm going to clarify the naming in a separate PR.